File: Java/loops/higher_number.java
package br.com.geekuniversity.secao07;

import java.util.Scanner;

public class Exercicio01 {

	public static void main(String[] args) {
		//Declaração
		int num, maior = 0;
		Scanner entrada = new Scanner(System.in);
		
		//Entrada
		do {
			
			System.out.println("Digite um número inteiro ou 0 (zero) para sair: ");
			num = entrada.nextInt();
			//Processamento
			if (num > maior) {
				maior = num;
			}
		} while (num != 0);
		//Saída
		System.out.println("Maior: " + maior);
		entrada.close();
	}
}

File: Python/loops/higher_number.py
# coding=utf8
# Declaração
maior = 0
# Entrada
num = int (input ("Digite um número qualquer ou 0 (zero) para Sair: "))
while num != 0:
    if num > maior:
        maior = num
    num = int (input("Digite um número qualquer ou 0 (zero) para Sair: "))
print("Maior número: {0}".format(maior))